app/plugins/tutorial/tutorial.js

```diff
@@ -170,12 +170,12 @@ tutorialPlugin.run(function($window, $location, NavigationService, $translatePar
 
     $translatePartialLoader.addPart('./plugins/tutorial');
     $translate.refresh();
-    var iconClass = "fa fa-info-circle";
+    var iconSrc = "/plugins/tutorial/icons/icon-getstarted.svg";
```

Comment:
here you use an abs path,
in common.js you use a rel path, though.
does the abs path also work if admin ui is served from a different location than `/`?
```js
NavigationService.addNavBarElement("images/icons/icon-overview.svg", $filter('translate', 'NAVIGATION.OVERVIEW'), "/");
```